tests/dicts/test_benedict.py

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
-
```

Comment:
Generally a blank line is used after the encoding declaration, btw this will be dropped soon as no more needed.